docs/CONTRIBUTING.md

```diff
@@ -301,15 +301,13 @@ contributions) and contain a current email address.
 [Klipper-translations Project](https://github.com/Klipper3d/klipper-translations)
 is a project dedicated to translating Klipper to different languages.
 [Weblate](https://hosted.weblate.org/projects/klipper/) hosts all the
-Gettext strings for translating and reviewing. Locales can merge into
-the Klipper project once they satisfy the following requirements:
+Gettext strings for translating and reviewing. Locales can be displayed on
+<klipper3d.org> once they satisfy the following requirements:
```

Comment:
Using angle brackets in markdown (outside of backticks) causes havoc.  Use backticks, bold (via *), or underline (via _) instead.